nailgun/entities.py

```diff
@@ -1799,6 +1802,16 @@ def __init__(self, server_config=None, **kwargs):
         self._meta = {'api_path': 'api/v2/hostgroups', 'server_modes': ('sat')}
         super(HostGroup, self).__init__(server_config, **kwargs)
 
+    def create(self, create_missing=None):
+        """Do extra work to fetch a complete set of attributes for this entity.
+        For more information, see `Bugzilla #1223540
+        <https://bugzilla.redhat.com/show_bug.cgi?id=1223540>`_.
```

Comment:
This bug actually references compute resources, not host groups. Can you drop this bit from the docstring and file a github issue saying "file a BZ for `HostGroup.create` and link to it"?